app/views/Dashboard/Overview/PercentageCardGroup/index.tsx

```diff
@@ -477,7 +478,10 @@ function PercentageCardGroup(props: Props) {
                 regionTotalCase.indicatorValue,
                 regionTotalCase.errorMargin,
             );
-            const range = `${negativeRange} - ${positiveRange}`;
+
+            const range = regionTotalCase?.format === 'percent'
+                ? `[${negativeRange}% - ${positiveRange}%]`
```

Comment:
I think this contradicts with the requirement of showing the values up to the precision of one decimal value and complete value in tooltips. Can you check if this behaviour is okay?